.travis/debuild_test

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -ex
+
+# this script runs as root under docker 
+
+# Prepare for build
+export DEBIAN_FRONTEND=noninteractive
+export LC_ALL=C
+apt-get update
+apt-get -y dist-upgrade
+apt-get -y install devscripts build-essential lintian gdebi-core git
+
+DEB_VERS=$(LC_ALL=C dpkg-parsechangelog | grep '^Version:' | cut -d ' ' -f 2,2)
+PLAIN_VERS=$(echo ${DEB_VERS} | sed 's#-.*##')
+git archive --format=tar HEAD | gzip > ../singularity-container_${PLAIN_VERS}.orig.tar.gz
+
+# Install dependencies
+mk-build-deps
+mv singularity-container-build-deps_*.deb ..
+gdebi -n ../singularity-container-build-deps_*.deb
+
+# Build package
+debuild -us -uc
+
+echo "Directory containing the build .deb-files:"
+ls -la ..
+gdebi -n ../singularity-container_*.deb
```

Comment:
Instead of adding `gdebi` ... could you just use `dpkg -i` to install the `.deb` file?

Reply:
`dpkg` does not resolve dependencies, while `gdebi` does. So it's essential to use `gdebi` here to check that the dependencies expressed in the packaging scripts are actually complete.

Reply:
> dpkg does not resolve dependencies, while gdebi does.

Or rather: `dpkg` _does_ resolve dependencies, of course, but does not install them (it only complains if things are not installed yet) :wink:.

Reply:
I just found that you can apparently do `dpkg -i pkg.deb` followed by `apt-get -fy install` [1]. Will try this later.

[1] https://superuser.com/a/537371

Reply:
If you're going to do that, couldn't you do:

```
apt-get install ../singularity-container_*.deb
```

And then let `apt-get` pull the dependencies?

Reply:
Indeed, both of these should also work - of course `apt-get -fy install` may do more than wanted / expected, and `apt-get install ../singularity-container_*.deb` is an officially undocumented feature. The man-page says `Each package is a package name, not a fully qualified filename` when talking about parameters to `install`. So this unofficial feature may be broken in any future version of `apt-get`.

Reply:
Hrmm.... If that's the case, then I guess I would stick with installing `gdebi` and using it, since the `apt-get -fy install` may have unintended consequences, and `apt-get install package_name.deb` is a unsupported "feature."

I would have just thought apt-get could install from a local `.deb` package file as if it was pulling from sources.list, and not thought anything of it.